src/NuGet.Core/NuGet.Build.Tasks/RestoreTask.cs

```diff
@@ -23,7 +23,12 @@ namespace NuGet.Build.Tasks
     /// </summary>
     public class RestoreTask : Microsoft.Build.Utilities.Task, ICancelableTask, IDisposable
     {
-        private const string HttpUserAgent = "NuGet MSBuild Task";
+#if IS_DESKTOP
```

Comment:
nvm, i found it [here](https://github.com/NuGet/NuGet.Client/blob/e6e7d4ddb2499f2cce370f9f1c477d204aec90ba/build/common.targets#L5).